app/tests/integration/nimbus/conftest.py

```diff
@@ -233,7 +233,7 @@ def default_data(application, experiment_name, load_experiment_outcomes):
 
 @pytest.fixture
 def create_experiment(base_url, default_data):
-    def _create_experiment(selenium):
+    def _create_experiment(selenium, isRollout=False):
```

Comment:
Could you "pythonify" this please 😁 but yes, this is the exact pattern I was referencing!

Reply:
yes I can! 🙌